python/singa/autograd.py

```diff
@@ -614,10 +682,28 @@ def __call__(self, x):
             y = add_bias(y, self.b, axis=0)
         return y
 
+    def get_params(self):
+        if self.bias:
+            return {'W': self.W, 'b': self.b}
+        else:
+            return {'W': self.W}
+
+    def set_params(self, **parameters):
```

Comment:
will this function be called? or set_one_param() is enough?

Reply:
no, I will not use this function